docs/developer/index.rst

```diff
@@ -18,6 +18,8 @@ the `tutorials <http://tutorials.kytos.io/>`__ made by Kytos Team.
 
 .. include:: api.rst
 
+.. include:: hacking.rst
+
```

Comment:
I would also add `hacking` prior to the `api` and the `listened_events`. It makes more sense, doesn't it?